samples/tiled-convolution/tiled-convolution.cpp

```diff
@@ -160,25 +208,22 @@ int main() {
   // mask array
   std::vector<data_t> filter(fil_size.size(), filter_data);
 
+  // enabling SYCL queue profiling
   auto property_list =
       cl::sycl::property_list{cl::sycl::property::queue::enable_profiling()};
-  // constructing a SYCL queue for OpenCL device where
-  // automatically build the underlying context and command_queue for the
-  // chosen device.
+
+  auto selector = cl::sycl::default_selector();
   auto sycl_queue = cl::sycl::queue(
-      (cl::sycl::default_selector()),
+      selector,
```

Comment:
What I was saying with this comment is that you shouldn't need the selector at all, you can remove this and line 215, according to the spec.